### DISCUSSION TITLE
How to place SEO meta tags such as title, description, canonical, etc. at the beginning of the <head> html tag

### DISCUSSION THREAD

**Discussion by steadyGuy** [vercel/next.js#56875] [Help]
### Summary

Hello everyone. Is there any way to put SEO meta tags using _generateMetadata_ functions at the beginning of the `<head>`?
This is a requirement from our SEO specialist, who says that placing them near the top of the `<head>` section is recommended because search engines may give more weight to the text in the title if it appears early in the HTML structure.

Is it possible to do this without interfering with the next.js package code?

Currently, I have different links (font preload, CSS files, etc.) and scripts preloading before meta tags.
![image](https://github.com/vercel/next.js/assets/41287687/5ba9cb4c-3f5a-4e14-9f71-426852d1e214)

It was possible to do it in Next.js 12 with the `<Head>` component. But with _generateMetadata_, I can't find any way to overcome it.

**Comment by dlehmhus** [comment]
@steadyGuy did you find any solution to this? I'm facing quite similar requirements at my current project.
